cli/utilities/utils.py

```diff
@@ -452,3 +452,26 @@ def get_lvm_on_osd_container(container_id, node, format="json"):
     cmd = f"ceph-volume lvm list --format {format}"
     out, _ = exec_command_on_container(node=node, ctr=container_id, cmd=cmd, sudo=True)
     return loads(out)
+
+
+def get_disk_devlinks(node, disk):
+    """
+    Parses the devlink data to identify the by-id and by-path values
+    Args:
+        node (ceph): Node to execute cmd
+        disk (str): disk nam. e.g: /dev/vdb
+
+    Returns (list, list): by-id and by-path values
+    """
+
+    cmd = f"udevadm info --query=property {disk} | grep DEVLINKS"
```

Comment:
better way to write -
```
    cmd = f"udevadm info --query=property {disk} | grep DEVLINKS"
    out, _ = node.exec_command(cmd=cmd)
    id, path = [], []
    for entry in out.split(" "):
        path = entry.split("=")[-1]
        if "by-id" in path:
            id.append(path)
            continue

        path.append(path)

    return id, path
```

Reply:
One doubt here Vaibhav, kept it in if loops so that we only process cases which are required to us, i.e by-id and by-path. In this suggestion, all other cases will also be appended to the result, if any, right Vaibhav? In that case, can we keep it simple with if checks as it is now?

Reply:
Okay. So we are expecting values other than `by-id` and `by-path` ?

Reply:
Yes Vaibhav, one example is 
```
DEVLINKS=/dev/block/8:48 /dev/disk/by-id/scsi-360001ff00d606eca050008000082fe45 /dev/disk/by-path/pci-0000:07:00.1-fc-0x28000001ff0414e2-lun-5 /dev/disk/by-uuid/ec5eb4a9-7189-4dba-939a-776f6b49748d /dev/disk/by-id/wwn-0x60001ff00d606eca050008000082fe45```

So there is by-uuid, block etc along with by-id and by-path.